packages/@aws-cdk/aws-apigatewayv2-authorizers/test/http/integ.lambda.ts

```diff
@@ -0,0 +1,42 @@
+import * as path from 'path';
+import { HttpApi, HttpMethod } from '@aws-cdk/aws-apigatewayv2';
+import { LambdaProxyIntegration } from '@aws-cdk/aws-apigatewayv2-integrations';
+import * as lambda from '@aws-cdk/aws-lambda';
+import { App, Stack } from '@aws-cdk/core';
+import { HttpLambdaAuthorizer, HttpLambdaResponseType } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * * `curl --request --header 'X-API-Key: 123' GET <url>` should return 200
```

Comment:
I gave this a shot but didn't get 200. Can you verify?

Reply:
Did you get a chance to look at this?

Reply:
Fixed in latest